R/build-home-authors.R

```diff
@@ -105,22 +105,56 @@ author_list <- function(x, authors_info, comment = FALSE) {
   roles <- paste0(role_lookup[x$role], collapse = ", ")
   substr(roles, 1, 1) <- toupper(substr(roles, 1, 1))
 
+  orcid <- x$comment %>% purrr::pluck('ORCID')
+  x <- remove_orcid_comment(x)
+
   list(
     name = name,
     roles = roles,
-    comment = x$comment
+    comment = x$comment,
+    orcid = orcid_link(orcid)
   )
 }
 
 author_desc <- function(x, comment = TRUE) {
   paste(
     x$name,
     "<br />\n<small class = 'roles'>", x$roles, "</small>",
+    if (!is.null(x$orcid))
+      x$orcid,
     if (comment && !is.null(x$comment))
       paste0("<br/>\n<small>(", x$comment, ")</small>")
   )
 }
 
+remove_orcid_comment <- function(x) {
+
+  if ("ORCID" %in% names(x$comment)) {
+    if (length(x$comment) == 1) {
+      x$comment <- list(NULL)
+    } else {
+      xx <- x$comment[!names(x$comment) == "ORCID"]
+      if (length(xx) == 0 || is.null(xx)) {
+        x$comment <- list(NULL)
+      } else {
+        x$comment <- xx
+      }
+    }
+  }
+
+  x
+}
+
+orcid_link <- function(orcid) {
+  if (is.null(orcid)) return(NULL)
+
+  paste0(
+    "<a href='https://orcid.org/", orcid,
+    "' target='orcid.widget' rel='noopener noreferrer' style='vertical-align:top;'>",
```

Comment:
I think we can drop the `rel` and `style`, and maybe even the `img` in favour of doing this with css?